static/js/components/GroupSources.jsx

```diff
@@ -504,7 +510,7 @@ GroupSourcesTable.propTypes = {
       dec: PropTypes.number,
       alias: PropTypes.string,
       redshift: PropTypes.number,
-      classifications: PropTypes.arrayOf(PropTypes.string),
+      classifications: PropTypes.arrayOf(PropTypes.object),
```

Comment:
This should fail on the linter; arrays of objects should specify the shape of that object.

Reply:
I hope this is the right way to do this...